lib/main.js

```diff
@@ -120,6 +121,15 @@ module.exports.runBridge = Promise.coroutine(function*(port, config, reg) {
     }
 
     // run the bridge
-    var ircBridge = new IrcBridge(config, reg);
+    ircBridge = new IrcBridge(config, reg);
```

Comment:
We should absolutely be guarding against clobbering the `ircBridge` global. Either immediately die if one already exists, or whine loudly in the logs, up to you.